MAINTAINER_README.md

```diff
@@ -12,8 +12,7 @@ Quicksprint uses [DDEV-Local](https://github.com/drud/ddev), docker, and a clone
 * To create a sprint package, run the script package_drupal_script.sh. (You can also retrieve a tarball or zipball of the results at https://github.com/drud/quicksprint/releases instead of doing this step.)
 * Your users will then download and unarchive the tarball or zipball.
 * Linux and Mac users run install_ddev.sh, Windows users run install_ddev.cmd inside the directory created by the tarball.
-* After installation, Linux and Mac users can start up an instance by cd'ing to ~/Sites/sprint and running start_sprint.sh,  Windows users run start_sprint.cmd in Sites/sprint of their user folder.
-* _Windows users must run an additional command_:  the start_clean command will tell you what the command is.
-
-At this point the plain vanilla git-checked-out drupal8 HEAD version should be running at http://drupal8.ddev.local 
+* After installation, Linux and Mac users can start up an instance by cd'ing to ~/Sites/sprint and running start_sprint.sh. Windows users run start_sprint.cmd in Sites/sprint of their user folder.
+* _Windows users must run an additional command_: the start_clean command will tell you what the command is.
 
+At this point the plain vanilla git-checked-out drupal8 HEAD version should be running at http://drupal8.ddev.local.
```

Comment:
I do need to verify this one thing because, if my memory serves me correctly, there was an index/number appended to the project versus a straight 'drupal8'. Let's merge for now and we can correct. I'm about to retest, but this is the only thing that jumped out at me as something that needed a doublecheck.